src/target/renesas.c

```diff
@@ -281,16 +378,19 @@ typedef enum {
 #define RV40_FCPSR_ESUSPMD 1U
 
 static bool renesas_uid(target_s *t, int argc, const char **argv);
+static bool renesas_cmd_option(target_s *target, int argc, const char **argv);
 
 const command_s renesas_cmd_list[] = {
 	{"uid", renesas_uid, "Prints unique id"},
+	{"option", renesas_cmd_option, "Manipulate option settings"},
 	{NULL, NULL, NULL},
 };
 
 typedef struct renesas_priv {
 	uint8_t pnr[17]; /* 16-byte PNR + 1-byte null termination */
 	renesas_pnr_series_e series;
 	target_addr_t flash_root_table; /* if applicable */
+	renesas_family_s details;
```

Comment:
now that `renesas_family_s` also holds the `renesas_pnr_series_e` we are storing redundant information, we can remove `series` here and refactor the code to use the `details`

Reply:
agree, at this stage we can flatten `renesas_pnr_series_e` completely and remove `details` too

Reply:
agreed